app_route/outputs.tf

```diff
@@ -5,3 +5,7 @@ output "endpoint" {
 output "route_id" {
   value = cloudfoundry_route.app_route.id
 }
+
+output "host" {
+  value = cloudfoundry_route.app_route.host
+}
```

Comment:
[suggestion] I'm not opposed to including this, but I don't think it's necessary to add